### DISCUSSION TITLE
TypeError: 'VoiceClient' object is not iterable

### DISCUSSION THREAD

**Discussion by mrracoonman** [Rapptz/discord.py#6065] [Q&A] [answered]
When I try to run this within my code:

    voice = get(ctx.guild.voice_client, guild=ctx.guild)
    ydl_opts = {
        'format': 'bestaudio/best',
        'postprocessors': ({
            'key': 'FFmpegExtractAudio',
            "preferredcodec": 'mp3',
            'prefferedquality': '192',

        }),
    }

It sends me this error:

TypeError: 'VoiceClient' object is not iterable

And I've searched but I cant figure out why I'm getting it. If I'm missing something or am doing something wrong, could it please be pointed out?

Win 10, py 3.8.6

**Comment by Rapptz** [comment, marked as answer]
`discord.utils.get` expects an iterable (something you can use a `for` loop with) as its first argument. A `VoiceClient` cannot be iterated with so you get an error. I am unsure what you're planning on doing there, but you probably just want to use `ctx.guild.voice_client` and not do the `get` call at all.
